### PR TITLE
strconv: v_printf documentation

### PR DESCRIPTION
**Pull request by penguindark**
**What's inside**
- first release of the `v_printf` documentation

### CODE REVIEW

**Comment by JalonSolov** [comment]
How would you feel about suggestions on English grammar?  I don't want to offend or be rude.

**Comment by penguindark** [comment]
Please, change all the grammar you think is needed,
my english is very poor :)

File: vlib/strconv/format.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+The width field may be omitted, or a numeric integer value, or a dynamic value when passed as another argument when indicated by an asterisk `*`. For example, `v_printf("%*.s", 5, my_string)` will result in `   mystring` being printed, with a total width of 5 characters.
+
+#### Length field 
+
+The Length field can be omitted or be any of:
+
+| Character | Description                                                  |
+| --------- | ------------------------------------------------------------ |
+| `hh`      | For integer types, causes `printf` to expect an `byte`or `i8` argument. |
```
Comment (JalonSolov) [inline vlib/strconv/format.md:58]: Technically is should be "expect an" in all 4 lines in the table, as it is an article describing 'argument'.  "`byte` or `i8`" are extra words to narrow down the possibilities.

Also, make sure there is a space before "or" on all 4 lines.

File: vlib/strconv/format.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+[                 -10][                 -10]|
+[                -100][                -100]|
+[               -1000][               -1000]|
+[              -10000][              -10000]|
+
+```
+
+## Utility functions
+
+In the format module are present some utility functions:
```
Comment (JalonSolov) [inline vlib/strconv/format.md:212]: "The format module also has some utility functions:"

**Comment by JalonSolov** [comment]
I am always amazed when someone says their English is not good, when I understood them perfectly.  :-)

My suggestions are here just to make it more proper English, not to try to change what you said.

**Comment by penguindark** [comment]
Thanks for the review :)
I'll correct the document this night, I hope to have some spare time to do it.
Thanks again.

**Comment by penguindark** [comment]
Other suggestions? :)

**Comment by JalonSolov** [comment]
Only one, that I didn't notice before.  At the end, you have "rm_tail_zeros".  It is usually "trailing" rather than "tail".  The descriptions would need the same change.

This would require changing the function name in the code as well, though.  Might not be worth the effort, it is just a little bit odd to a native English speaker.

Otherwise fantastic job!  Thanks for doing this.  :-)

**Comment by penguindark** [comment]
tnx to you :)
Now I'm short of time for modify the source code, but I will try in the future.

**Pull request merged**